tests/Aspire.MongoDB.Driver.Tests/ConformanceTests.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.ConformanceTests;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using MongoDB.Driver;
+using Xunit;
+
+namespace Aspire.MongoDB.Driver.Tests;
+
+public class ConformanceTests : ConformanceTests<IMongoClient, MongoDBSettings>
```

Comment:
Should override:

* SupportsKeyedRegistrations
* CanConnectToServer
* InvalidJsonToErrorMessage